src/database/index.tsx

```diff
@@ -166,3 +171,34 @@ export const getFacilities = async (): Promise<Array<Facility> | null> => {
     return null;
   }
 };
+
+export const createBaselineScenario = async () => {
```

Comment:
This is the only new code.  The stuff above is just renaming things and moving repetitive code into its own method.